src/global/tracking/CKFTracking_factory.cc

```diff
@@ -36,8 +29,8 @@ void eicrecon::CKFTracking_factory::Init() {
     InitLogger(app, param_prefix, "info");
 
     // Get ACTS context from ACTSGeo service
-    auto acts_service = app->GetService<ACTSGeo_service>();
-    auto dd4hp_service = app->GetService<DD4hep_service>();
+    auto acts_service   = app->GetService<ACTSGeo_service>();
+    auto dd4hep_service = app->GetService<DD4hep_service>();
```

Comment:
This is unused, right?
```suggestion
    auto dd4hep_service = app->GetService<DD4hep_service>();
```